packages/protocol/test/authorizers/ClientWhitelister.sol.js

```diff
@@ -0,0 +1,813 @@
+/* globals context ethers */
+
+const { expect } = require('chai');
+
+const AdminRank = {
+  Unauthorized: 0,
+  Admin: 1,
+  SuperAdmin: 2,
+};
+
+let roles;
+let selfClientRrpAuthorizer;
+let api3ClientRrpAuthorizer;
+let airnodeRrp;
+let airnodeId;
+const api3AdminnedEntity = ethers.constants.HashZero;
```

Comment:
Can we just use `airnodeId` wherever `api3AdminnedEntity` is being used? I think that closest to a real world value instead of using `HashZero`.

Reply:
So, the api3Authorizer will use bytes32(0) if I recall correctly which is why I used HashZero to represent that

Reply:
My bad and I like that you used the constant 👍🏻 . Resolving this comment.